packages/core/src/customizations/src/myAccount/extensions/orders/[id]/after.tsx

```diff
@@ -0,0 +1,7 @@
+// placeholder for the after component
+
+function After() {
+  return <></>
+}
+
+export default After
```

Comment:
```suggestion
// placeholder for the after component

function After() {
  return <></>
}

export default After
```

`question:` Do we need this placeholders already created here and also need to create new ones on the starter for it to work?

https://github.com/vtex-sites/faststoreqa.store/pull/773/files#diff-46551ae41e9bce034d7aacd01a57447155dea49e6636616054e8aa7e6d342de5

Reply:
Yes, this one works as a fallback in the case there is no custom one from the store.